api/v1/performanceprofile_types.go

```diff
@@ -70,6 +70,9 @@ type PerformanceProfileSpec struct {
 	// Defaults to "false"
 	// +optional
 	GloballyDisableIrqLoadBalancing *bool `json:"globallyDisableIrqLoadBalancing,omitempty"`
+	// WorkloadHints defines hints for different types of workloads. It will allow defining exact set of tuned and
+	// kernel arguments that should be applied on top of the node.
+	WorkloadHints *WorkloadHints `json:"workloadHints,omitempty"`
```

Comment:
I wonder if this warrants a API bump. Probably not, but worth checking. Also, we need to check we're doing OK with the API conversion rules (v2 <-> v1).
The API per se looks nice.

Reply:
yep, we should have v1 and v2 API consistent because of OpenShift rules.